### DISCUSSION TITLE
Why are same attachments duplicated -1 -2 -3, etc. some as high as -30?

### DISCUSSION THREAD

**Discussion by cma1111** [RealRaven2000/FiltaQuilla#336] [Announcements]
Hello and used to be a Eudora user and loved it due to the ability to always download attachments.  This product was my life saver but it is eating up gigs of my harddrive space due to downloading the same attachment many times.  Grant you  not always every attachment but about 60% of the time.

What should I do to stop this?

I have a simple rule, match all messages and then save attachments to....

Windows 7 x64, latest TB

Thanks in advance,

Allen

**Comment by RealRaven2000** [comment]
> Hello and used to be a Eudora user and loved it due to the ability to always download attachments. This product was my life saver but it is eating up gigs of my harddrive space due to downloading the same attachment many times. Grant you not always every attachment but about 60% of the time.
> 
> What should I do to stop this?
> 
> I have a simple rule, match all messages and then save attachments to....

In don't know what causes the same filter to run again and again but I would move the email to another folder or tag them and exclude the tag so running the same filter will be avoided. Once Thunderbird decides that the filter needs to be run again (and obviously your same conditions still apply), Filtaquilla has no way of knowing that the original attachment was already saved before (matching the name is not sufficient).

**Reply by xLorix** [reply]
I experienced a similar issue with an IMAP account. I think the duplication of saved attachments occured every time I restarted Thunderbird (maybe after crash?) or for some reason the IMAP mailbox was reloaded, the inbox messages filters were re-processed and the attachments saved again with a "-n" suffix.
And yes, the solution for me was to add an action at the end of the filter to move the messages out of inbox, so they would not be re-processed.
Bye,
Loris
